mllm/models/internlm2/modeling_internlm2.hpp

```
@@ -0,0 +1,324 @@
+
+    auto inv_freq = makeRoPEInvFreq(cfg.head_dim, cfg.rope_theta, rope_linear_scale_);
+    registerBuffer("inv_freq_base", inv_freq);
+  }
+
+  ARGenerationOutputPast forward(const ARGenerationOutputPast& input, const ARGenerationArgs& /*args*/) override {
+    auto sequence = input.at("sequence");
+
+    auto batch_size = sequence.shape()[0];
+    auto seq_len = sequence.shape()[1];
+
+    Tensor position_ids;
```

Clang (14.0.6)

[error] variable 'position_ids' is not initialized

(cppcoreguidelines-init-variables,-warnings-as-errors)